module/applications/sheets/item-taxonomy-sheet.mjs

```diff
@@ -74,6 +81,27 @@ export default class CrucibleTaxonomyItemSheet extends CrucibleActorDetailsItemS
 
   /* -------------------------------------------- */
 
+  /**
+   * Update the resistance numerical inputs when a change is made to selected immunities,
+   * disabling the corresponding inputs and overriding their visual value as 4.
+   * @param {string[]} selected The ids of the damage types which are currently selected as immunities
+   */
+  #handleImmunities(selected) {
+    const resistances = this.element.querySelector(".resistances");
+    const inputs = resistances.querySelectorAll("input[type=number]");
+    for ( const input of inputs ) {
+      if ( selected.includes(input.dataset.damageType) ) {
+        input.disabled = true;
+        input.value = 4;
```

Comment:
I'm thinking now that I'm seeing this, that if we are disabling the input regardless, maybe we can visually improve it by swapping the input type to `"text"` and displaying the value as `&infin;`? Instead of displaying it as `4` which is not as intuitive.